public class Variables {
    static void main(String[] args) {
        //Task 3
        String a = new String("Wow");
        String c = "Wow";
        String b = a;
        String d = c;

        boolean b1 = a == b;
        boolean b2 = !d.equals(b + "!");
        boolean b3 = c.equals(a);

        if(b1 && b2 && b3){
            System.out.println("Success!");
        }
    }

}